pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu

```diff
@@ -145,6 +145,7 @@ RUN pip uninstall -y horovod \
  && ldconfig /usr/local/cuda-10.1/targets/x86_64-linux/lib/stubs \
  && HOROVOD_GPU_ALLREDUCE=NCCL HOROVOD_CUDA_HOME=/usr/local/cuda-10.1 HOROVOD_WITH_PYTORCH=1 pip install --no-cache-dir horovod==${HOROVOD_VERSION} \
  && ldconfig
+ && pip install mpi4py==3.0.3
```

Comment:
`&& ldconfig` requires ` \` after it for the rest of the multi-line RUN statement to work.

Reply:
Addressed this change across all the dockerfiles